uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-assignment.controller.spec.js

```diff
@@ -34,6 +34,19 @@ define([
       angular.extend(scope, Mock);
     }));
 
+    describe('on init', function () {
+      var resolvedData = { foo: 'foo', bar: 'bar' };
+
+      beforeEach(function () {
+        initController(resolvedData);
```

Comment:
Please do deepClone here. So then you don't need to care about mutation and also about testing.